### DISCUSSION TITLE
Copy pages across spaces

### DISCUSSION THREAD

**Discussion by OtherToday** [GitbookIO/community#145] [Feature Requests]
### What's your feature/idea?

I use Gitbook for course materials. I create spaces for each module and would like to copy content over into new pages. At the moment the only way to do this is to duplicate a space but this does not work if you need to merge the content from multiple spaces. 

- Web capture does not work on gitbook
- github sync removes rich media and gitbook formating (tabs etc)
- copy and paste aslo doesn't work

If you can't merge content, being able to export markdown from pages would help. 

Thanks

### What's your desired outcome?

Ability to duplicate and move pages between spaces. 

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hey @OtherToday!

Thank you so much for writing this feature request. I'm pleased to let you know that this is something our team is working on.

Just so you know, this idea was previously proposed in this thread:  https://github.com/orgs/GitbookIO/discussions/21 🙂.

I'll be closing this request in favor of the existing one.
